suite-native/module-accounts/src/components/AccountDetailScreenHeader.tsx

```diff
@@ -43,24 +45,19 @@ export const AccountDetailScreenHeader = ({
 
     return (
         <ScreenHeader
-            leftIcon={
-                <IconButton
-                    colorScheme="tertiaryElevation0"
-                    size="medium"
-                    iconName="chevronLeft"
-                    onPress={() => navigation.goBack()}
-                />
-            }
+            hasGoBackIcon
             rightIcon={
-                <IconButton
-                    colorScheme="tertiaryElevation0"
-                    size="medium"
-                    iconName="settings"
-                    onPress={handleSettingsNavigation}
-                />
+                !tokenName && (
+                    <IconButton
+                        colorScheme="tertiaryElevation0"
+                        size="medium"
+                        iconName="settings"
+                        onPress={handleSettingsNavigation}
+                    />
+                )
             }
             style={applyStyle(headerStyle)}
-            title={accountLabel}
+            title={`${accountLabel} ${tokenName}`}
```

Comment:
Please create some extra variable this like `const title = isToken ? "title 1" : "title 2"` because if you will do this it will add empty space after account label for non-tokens accounts which will cause title to be little bit of center.

Reply:
fixed in https://github.com/trezor/trezor-suite/commit/4d5c05278740cdd3d9e85981c1257cf8feac0a25